### PR TITLE
Changed network

### PR DESCRIPTION
**Pull request by LiorAgnin**
## Proposed changes

- Changed network to the new one.

## Types of changes

What types of changes does your code introduce?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)

### CODE REVIEW

File: server/src/utils/web3.js [hunk 1]
```diff
@@ -47,7 +47,7 @@ const send = async ({ web3, bridgeType, address }, method, options) => {
     const methodName = getMethodName(method)
     const nonce = account.nonces[bridgeType]
     console.log(`[${bridgeType}] sending method ${methodName} from ${from} with nonce ${nonce}. gas price: ${gasPrice}, gas limit: ${gas}`)
-    receipt = await method.send({ gasPrice, ...options, gas, nonce: nonce, chainId: bridgeType === 'home' ? 121 : undefined })
+    receipt = await method.send({ gasPrice, ...options, gas, nonce: nonce, chainId: bridgeType === 'home' ? 122 : undefined })
```
Comment (leonprou) [inline server/src/utils/web3.js]: can you put 122 into enum/config?

Comment (LiorAgnin) [reply server/src/utils/web3.js]: Done

File: dapp/src/utils/network.js [hunk 2]
```diff
@@ -1,18 +1,19 @@
...
-  fuse: 'https://explorer.fuse.io'
+  fuse: 'https://explorer.fusenet.io'
 }
 
 export const getBlockExplorerUrl = (networkType) => blockExplorers[networkType]
 
 export const getApiRoot = (networkType) => CONFIG.api.url[networkType]
 
-export const isFuse = (provider) => provider.connection.networkVersion === '121'
+export const isFuse = (provider) => provider.connection.networkVersion === String(invert(networkIdToName)['fuse'])
```
Comment (leonprou) [inline dapp/src/utils/network.js]: that kinda comlicate. let's just add the chainId to config file. then we'll use the config here. and define `networkIdToName` as:
networkIdToName = 
{
 ....
 [CONFIG...fuse]: fuse
 ...
}

File: server/config/default.js [hunk 3]
```diff
@@ -34,21 +34,20 @@ module.exports = {
     home: {
       name: 'fuse',
       bridgeType: 'home',
+      chainId: 122,
```
Comment (leonprou) [inline server/config/default.js]: you can do the same for the dapp client

**Pull request merged**